pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/measurements/MeasurementsTNCuda.hpp

```diff
@@ -164,6 +165,97 @@ template <class TensorNetT> class MeasurementsTNCuda {
         return h_res;
     }
 
+    /**
+     * @brief Utility method for samples.
+     *
+     * @param wires Wires can be a subset or the full system.
+     * @param num_samples Number of samples
+     * @param numHyperSamples Number of hyper samples to use in the calculation
+     * and is default as 1.
+     *
+     * @return std::vector<std::size_t> A 1-d array storing the samples.
+     * Each sample has a length equal to the number of qubits. Each sample can
+     * be accessed using the stride sample_id*num_qubits, where sample_id is a
+     * number between 0 and num_samples-1.
```

Comment:
Let's use backtick for formulas everywhere.
```suggestion
     * @return std::vector<std::size_t> A 1-d array storing the samples.
     * Each sample has a length equal to the number of qubits. Each sample can
     * be accessed using the stride `sample_id * num_qubits`, where `sample_id` is a
     * number between `0 and `num_samples - 1`.
```

Reply:
We should also replace `num_qubits` with the size of given wires. right?

Reply:
Done!